README.md

```diff
@@ -4,6 +4,15 @@ Python bindings for the Domino API.
 
 Permits interaction with a Domino deployment from Python using the [Domino API](https://dominodatalab.github.io/api-docs/).
 
+## Version Compatibility Matrix
+
+`python-domino` library is compatible with different versions of the `Domino`:
+
+| Domino Versions | Python-Domino |
+| --------------- |:-------------:|
+| 3.6.x or Lower  | 0.3.5         |
+| 4.0.x or Higher | 1.0.0         |
```

Comment:
We're releasing new version for 4.2, so probably should be:

4.1.x or Lower  0.3.5
4.2 or Higher   1.0.0